pkg/error/wrap.go

```diff
@@ -0,0 +1,74 @@
+package error
+
+import (
+	"fmt"
+	"runtime"
+	"strings"
+)
+
+//
+// Wrap an error.
+// Returns `err` when err is `nil` or *Error.
+func Wrap(err error) error {
+	if err == nil {
+		return err
+	}
+	if le, cast := err.(*Error); cast {
+		return le
+	}
+	bfr := make([]uintptr, 50)
+	n := runtime.Callers(2, bfr[:])
+	frames := runtime.CallersFrames(bfr[:n])
+	stack := []string{""}
```

Comment:
Is it desirable to have the first index in this slice be an empty string, or would it make sense to just allocate an empty slice here?

Reply:
The purpose of the empty string is to ensure the 1st stack frame is prefixed by the `\n` separator in Error.Stack().